src/dmd/opover.d

```diff
@@ -432,6 +432,43 @@ Expression op_overload(Expression e, Scope* sc, TOK* pop = null)
             }
         }
 
+        override void visit(PostExp pe)
+        {
+            AggregateDeclaration ad = isAggregate(pe.e1.type);
+            if (!ad)
+                return visit(cast(BinExp) pe);
+            auto fdr = search_function(ad, Id.opUnaryRight);
+            if (!fdr)
+                return visit(cast(BinExp) pe);
+
+            void errorRet()
+            {
+                result = ErrorExp.get();
+            }
+
+            StorageClass stc;
+            if (TemplateDeclaration d = fdr.isTemplateDeclaration())
+            {
+                if (FuncDeclaration fdr2 = d.onemember.isFuncDeclaration())
+                    stc = fdr2.storage_class;
+                else
+                {
+                    d.onemember.error("isn't a function");
+                    return errorRet();
+                }
+            }
+            else
+            {
+                fdr.error("isn't a template");
+                return errorRet();
+            }
+            if (stc & STC.disable)
```

Comment:
`stc` is declared on line 449, set on line 453, and used on line 465. This suggests a refactoring so it can be simply:
```
if (fdr2.storage_class & STC.disable)
```
and `stc` is eliminated.

Reply:
n.b. `frd2` is not in scope here. you may be confused by `fdr`declared on line 440 which is, but is the `opUnary` not the `opUnaryRight` (which should be `@disabled`) that is proposed by this PR